includes/Plugin.php

```diff
@@ -73,6 +73,13 @@ class Plugin {
 	 */
 	public $dashboard;
 
+	/**
+	 * Settings.
+	 *
+	 * @var Settings
+	 */
+	public $settings;
```

Comment:
The `Settings` class does not really expose anything useful, so maybe not expose it here for now either?

Reply:
It is useful to get the setting name. 

`$plugin->settings:: SETTING_NAME`

Reply:
That's gonna make it hard for us to maintain backwards compatibility though. But well, we can revisit before stable.